java/src/main/java/org/rocksdb/OptimisticTransactionDBOptions.java

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+package org.rocksdb;
+
+/**
+ * Used to control the Optimistic Concurrency Control (OCC) aspect of a database.
+ * It will be used during the creation of a
+ * {@link org.rocksdb.OptimisticTransactionDB} (i.e., OptimisticTransactionDB.open()).
+ * <p>
+ * As a descendent of {@link AbstractNativeReference}, this class is {@link AutoCloseable}
+ * and will be automatically released if opened in the preamble of a try with resources block.
+ */
+public class OptimisticTransactionDBOptions extends RocksObject {
+  public OptimisticTransactionDBOptions() {
+    super(newOptimisticTransactionOptions());
+  }
+
+  /**
+   * Set OccValidationPolicy for this instance.
+   * See {@link OccValidationPolicy}.
+   *
+   * @param policy The type of OCC Validation Policy
+   *
+   * @return this OptimisticTransactionOptions instance
+   */
+  public OptimisticTransactionDBOptions setOccValidationPolicy(final OccValidationPolicy policy) {
+    assert (isOwningHandle());
+    setOccValidationPolicy(nativeHandle_, policy.getValue());
+    return this;
+  }
+
+  /**
+   * Get OccValidationPolicy for this instance.
+   * See {@link OccValidationPolicy}.
+   *
+   * @return The type of OCC Validation Policy used.
+   */
+  OccValidationPolicy getOccValidationPolicy() {
```

Comment:
Function scope should be "public" and not "package-private"

Reply:
Getters in RocksJava are typically not prefixed `get` - see other classes for the naming convention.